projects/vorbis/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER paul.l.kehrer@mail.com
+RUN apt-get update && apt-get install -y make autoconf automake libtool pkg-config
+RUN git clone https://git.xiph.org/ogg.git
+RUN git clone https://git.xiph.org/vorbis.git
+ADD decode_fuzzer.cc $SRC/
+ADD https://upload.wikimedia.org/wikipedia/commons/f/f9/Beep_example.ogg $SRC/sample.ogg
```

Comment:
The more samples you add the better. 
It doesn't need to be in the first commit.